WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -13,6 +13,32 @@ class DashboardPromptsCardCell: UICollectionViewCell, Reusable {
         }
     }
 
+    // This is public so it can be accessed from the BloggingPromptsFeatureDescriptionView.
+    lazy var cardFrameView: BlogDashboardCardFrameView = {
```

Comment:
Perhaps would be better to set this as read-only?
```suggestion
    private(set) lazy var cardFrameView: BlogDashboardCardFrameView = {
```

Reply:
Oh good catch! Thank you!